pkg/kubelet/metrics/metrics.go

```diff
@@ -211,6 +213,30 @@ var (
 		[]string{},
 	)
 
+	// FirstNetworkPodStartSLIDuration is a gauge that tracks the duration (in seconds) it takes for the first network pod to run,
+	// excluding the time for image pulling. This metric should reflect the "Pod startup latency SLI" definition and measure the time
+	// for the CNI plugin to be able to provide the network capabilities to the Pod.
+	// ref: https://github.com/kubernetes/community/blob/master/sig-scalability/slos/pod_startup_latency.md
+	FirstNetworkPodStartSLIDuration = metrics.NewGauge(
```

Comment:
@aojea - I think you were chatting with @logicalhan about it already and I don't know where exactly you converged, but I wanted to throw one more idea and hear why it bad.

What if instead introducing a new metrics for that, we will introduce a label for the already existing PodStartSLIDuration metrics like "kind"? [better name is needed].

Then we will be able to distinguish different types of pods in that, e.g.:
"kind=first-pod-with-network"
for this particular case.

But I think it would be universally useful, because we will certainly need this for other cases, e.g. we want to be able to distinguish stateless and stateful pods because they clearly have a very different performance characteristic. And we would be able to to reuse the same label for it, e.g.:
kind=stateless
kind=stateful

@logicalhan @dgrisonnet - thoughts?

Reply:
That was actually what I originally proposed. However, given that this metric is a stopgap until we can properly pipe it via Status or something, I'm okay with an isolated metric which is of stability class `Internal` so that it doesn't get output in our auto generated docs and will be removed after a year or so.

Reply:
Being able to distinguish pods could be useful, but it don't think it would make sense to add the label for the current use case as we are essentially trying to measure something different. The fact that we are measuring it via a specific pod latency is an implementation detail, that I don't think we should surface in the general purpose metric.

But as Han mentioned, the general consensus is to introduce a new `Internal` metric as a temporary solution.

Reply:
> kind=stateless

Do we have that granularity on the kubelet?